common/script/index.coffee

```diff
@@ -931,9 +1005,18 @@ api.wrap = (user, main=true) ->
         user.items.quests[item.key] ?= 0
         user.items.quests[item.key] += 1
         user.stats.gp -= item.goldValue
+        analyticsData = {
+          uuid: user._id,
+          itemKey: item.key,
+          itemType: 'Market',
+          goldCost: item.goldValue,
+          acquireMethod: 'Gold',
+          category: 'behavior'
+        }
```

Comment:
@SabreCat Does this data look right to you? I added it in after merging in your changes on develop.